src/app/beer_garden/user.py

```diff
@@ -72,7 +72,9 @@ def initiate_user_sync() -> None:
     from beer_garden.router import route
 
     users = User.objects.all()
-    gardens = Garden.objects.filter(connection_type__nin=["LOCAL", None])
+    gardens = Garden.objects.filter(
+        connection_type__nin=["LOCAL", None], status="RUNNING"
```

Comment:
The `route` call seemed to fail if the garden wasn't running, resulting in a 500 error.  This ought not be the case, but I just decided to not attempt to route to gardens that aren't running.